docker-compose.yml

```diff
@@ -15,47 +15,29 @@
 ---
 version: '3'
 services:
-  zookeeper:
-    image: confluentinc/cp-zookeeper:5.1.0
+  kafka-zookeeper:
+    image: openzipkin/zipkin-kafka
+    # If using docker machine, uncomment the below and set your bootstrap
+    # server list to 192.168.99.100:19092
+    # environment:
+    # - KAFKA_ADVERTISED_HOST_NAME=192.168.99.100
     ports:
-    - 2181:2181
-    environment:
-      ZOOKEEPER_CLIENT_PORT: 2181
-      ZOOKEEPER_TICK_TIME: 2000
-  kafka:
-    image: confluentinc/cp-kafka:5.1.0
-    ports:
-    - 9092:9092
-    - 29092:29092
-    environment:
-      KAFKA_BROKER_ID: 1
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092,PLAINTEXT_HOST://localhost:29092
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
-      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
-    depends_on:
-      - zookeeper
-  kafka-manager:
-    image: syscomiddleware/kafka-manager:1.3.3
-    ports:
-      - 19000:9000
-    environment:
-      ZK_HOSTS: zookeeper:2181
+      - 2181:2181
+      - 9092:9092
+      # port 19092 is listening on localhost by default. In normal Docker,
+      # you can set your bootstrap server list to localhost:19092
+      - 19092:19092
   zipkin:
```

Comment:
I think you want to set the container names to match docker-zipkin so that they recreate and are not accidentally redundant etc.

```
± |develop ?:1 ✗| → docker ps
CONTAINER ID        IMAGE                     COMMAND                  CREATED             STATUS              PORTS                                                                      NAMES
7ff0e382172b        jeqo/zipkin-kafka         "/busybox/sh -c 'exe…"   10 minutes ago      Up 10 minutes       0.0.0.0:9411->9411/tcp                                                     zipkin-storage-kafka_zipkin_1
de0e6ef5d71a        openzipkin/zipkin-kafka   "/busybox/sh /kafka/…"   10 minutes ago      Up 10 minutes       0.0.0.0:2181->2181/tcp, 0.0.0.0:9092->9092/tcp, 0.0.0.0:19092->19092/tcp   zipkin-storage-kafka_kafka-zookeeper_1

```